WordPress/Classes/ViewRelated/Blaze Campaigns/BlazeCampaignsViewController.swift

```diff
@@ -234,10 +294,43 @@ extension BlazeCampaignsViewController: NoResultsViewControllerDelegate {
     }
 }
 
+// MARK: - Show / hide Promote button
+
+extension BlazeCampaignsViewController: Subscriber {
+
+    typealias Input = Bool
+    typealias Failure = Never
+
+    func receive(subscription: Subscription) {
+        subscription.request(.unlimited)
+    }
+
+    func receive(_ input: Bool) -> Subscribers.Demand {
+        let isHidden = input
+
+        guard promoteButton.isHidden != isHidden else {
+            return .unlimited
+        }
+
+        UIView.animate(withDuration: 0.3, delay: 0, options: [.curveEaseIn], animations: {
```

Comment:
I think it needs `.allowUserInteraction`. I'm not sure if there is an issue because I was testing in a simulator, but it appeared to be temporarily blocking the interactions.

Reply:
Do you mean user interaction is disabled while the view's animating? If so, that's the desired behavior imo.